catalog/spatial/registry/pom.xml

```diff
@@ -68,6 +68,11 @@
             </dependency>
             <dependency>
                 <groupId>org.codice.ddf.registry</groupId>
+                <artifactId>registry-federation-admin-api</artifactId>
```

Comment:
Looks like only registry-app is using this dependency, do we need to have it at this level?